styles/global.scss

```diff
@@ -0,0 +1,100 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@400;700&family=Barlow:wght@400;700&family=Bellefair&display=swap');
+
+// CSS RESET
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html:focus-within {
+  scroll-behavior: smooth;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+p,
+figure,
+blockquote,
+dl,
+dd {
+  margin: 0;
```

Comment:
This made the text-blocks on the artist-description collapse. But it's nice to move away from margins, so perhaps give the `<div class="text" id="text">` that holds the description-paragraphs `display: flex;`, `flex-direction: column;` and `gap: <some good size>;`. 

Maybe we should make some css-variables for sizes? i.e. xs, s, m, l, xl? Which we can use for gaps, paddings etc in the page? :)

Reply:
Eggsellent comment future husband of my life.